src/assets/getTextData.js

```diff
@@ -173,7 +173,7 @@ const getClientUrl = () => {
 const sendDataToClient = () => {
   const { selectionData, linkData } = getTextData(document.body);
   const iframeHeight = getPageHeight();
-  window.parent.postMessage({ selectionData, linkData, iframeHeight }, getClientUrl());
+  window.parent.postMessage({ key: 'selectionData', selectionData, linkData, iframeHeight }, getClientUrl());
```

Comment:
Instead of `key` lets use `type`. As in 'type of message'.

Also change instead of the string 'selectionData' something that is more accurate. I am thinking 'websiteData'